test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -913,6 +924,14 @@ UCS_TEST_P(test_ucp_sockaddr, ep_query) {
     ep_query();
 }
 
+UCS_TEST_P(test_ucp_sockaddr, set_local_sockaddr)
+{
+    if (m_test_addr.is_rdmacm_netdev()) {
+        listen_and_communicate(false, 0, true);
+        ep_query();
+    }
```

Comment:
pls use `UCS_TEST_SKIP_R` if you want to skip the test by some reason

Reply:
Please check the change.